pyuvdata/uvflag/uvflag.py

```diff
@@ -1054,6 +993,8 @@ def check(self, check_extra=True, run_check_acceptability=True):
         elif self.type == "antenna":
             if self.antenna_numbers is not None:
                 if not all(ant in self.antenna_numbers for ant in self.ant_array):
+                    print(self.antenna_numbers)
+                    print(np.unique(self.ant_array))
```

Comment:
Do we want these prints here? Maybe just state in the error message which antennas don't belong to both?

Reply:
Oops, I meant to take those out. But I think you're right, putting them in the error message is even better.